analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -235,9 +235,16 @@ abstract class PackageManager(
             }
         }
 
+        afterResolution(definitionFiles)
+
         return result
     }
 
+    /**
+     * Optional step to run after dependency resolution, like cleaning up temporary files.
+     */
+    protected open fun afterResolution(definitionFiles: List<File>) {}
```

Comment:
commit message: Why are you doing this? ..it is not needed.

I'd have the tendency to rather try to get rid of this 'before' function as it seems like an implementation detail and its exposure complicates the api without need. So this change is moving a bit into the opposite direction. On the other hand its consistent with what we have and makes it symmetric.

Reply:
The function will get used in an upcoming alternative Sbt package manager implementation. I just wanted to already push this for review now to make the upcoming PR smaller.

Reply:
Thanks!